pyqtgraph/widgets/PenSelectorDialog.py

```diff
@@ -0,0 +1,86 @@
+from pyqtgraph.Qt import QtCore, QtGui
+import PenSelectorDialogbox
+
+class PenSelectorDialog(QtGui.QDialog):
+    penChanged = QtCore.pyqtSignal(object)
+
+    def __init__(self, initialPen = None, parent = None):
+        QtGui.QDialog.__init__(self, parent)
+        self.ui = PenSelectorDialogbox.Ui_Dialog()
```

Comment:
Fix the naming to be specific
```suggestion
        self.ui = PenSelectorDialogbox()
```